compiler/make/make-runner.cpp

```diff
@@ -96,7 +96,12 @@ static int run_cmd(const string &cmd) {
   }
   argv.back() = nullptr;
 
+  #ifdef __APPLE__
+  int pid = fork();
```

Comment:
nit: it seems to me, that you can use **fork()** fo linux and other platforms as well